litellm/types/integrations/prometheus.py

```diff
@@ -219,23 +219,3 @@ class UserAPIKeyLabelValues(BaseModel):
     exception_class: Optional[str] = None
     status_code: Optional[str] = None
     fallback_model: Optional[str] = None
-
```

Comment:
we cannot delete this, can you find another work around @SmartManoj

Reply:
> we cannot delete this

Is there any unexpected side-effect?